app/src/test/java/org/mozilla/focus/MockitoExt.kt

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus
+
+import org.mockito.ArgumentCaptor
+import org.mockito.Mockito
+
+fun <T> any(): T = Mockito.any()
+
+inline fun <reified T> argumentCaptor(): ArgumentCaptor<T> = ArgumentCaptor.forClass(T::class.java)
```

Comment:
Let's not duplicate this code. This exists in the Android Components `support-test` module already. Let's include this module as a `testImplementation` dependency instead.

See: 
 - https://github.com/mozilla-mobile/android-components/blob/main/components/support/test/src/main/java/mozilla/components/support/test/KArgumentCaptor.kt
 - https://github.com/mozilla-mobile/android-components/blob/main/components/support/test/src/main/java/mozilla/components/support/test/Matchers.kt